toolchain/check/scope_stack.h

```diff
@@ -189,7 +191,7 @@ class ScopeStack {
 
  private:
   // An entry in scope_stack_.
-  struct ScopeStackEntry {
+  struct ScopeStackEntry : MoveOnly {
```

Comment:
Does the inheritance need to be non-public for this to work? If so, should that be noted on `MoveOnly`, perhaps with an example of how inheritance is intended?

Reply:
This is already public, because it's a struct. I can add the explicit `public` if you prefer, but given that we generally use structs in cases where we just want a collection of data, including `public` on a base class seems strange to me in the same way that including a `public:` access specifier in the body would seem strange.

But... could be that that's just me and my own peculiar perspective :) Would you like the `public` to be included explicitly here?

Reply:
Per discussion, some people may need to look it up because of familiarity with inheritance in the context of classes, rather than access control defaults. I certainly guessed wrong too. :)